web3.md

```diff
@@ -624,13 +624,13 @@ eth_sendTransaction
          <output> RD </output>
          <endPC> PCOUNT </endPC>
 
-    rule <k> _:String ~> #eth_sendTransaction_final => #rpcResponseError(-32000, "base fee exceeds gas limit") ... </k>
+    rule <k> _:String ~> #eth_sendTransaction_final => #popTxLists ~> #rpcResponseError(-32000, "base fee exceeds gas limit") ... </k>
```

Comment:
There's a rule above this:

```
    rule <k> (TXID:Int => "0x" +String #hashSignedTx(TN, TP, TG, TT, TV, TD, TW, TR, TS)) ~> #eth_sendTransaction_final ... </k>
         <message>
           <msgID> TXID </msgID>
           <txNonce>    TN </txNonce>
           <txGasPrice> TP </txGasPrice>
           <txGasLimit> TG </txGasLimit>
           <to>         TT </to>
           <value>      TV </value>
           <sigV>       TW </sigV>
           <sigR>       TR </sigR>
           <sigS>       TS </sigS>
           <data>       TD </data>
         </message>
```

I think this is a good place to pop TXID from. You can make another rule that matches the LHS of the k cell like this one that pops TXID from the tx lists, and this rule can match when TXID is not at the top of those lists.